File: 485-max-consecutive-ones/max-consecutive-ones.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        
        max=0
        count=0
        n=len(nums)

        for i in range(0,n):
            if(nums[i]==1):
                count+=1
            else:
                if(max<count):
                    max=count
                count=0
        
        if(max<count):
            max=count
        
        return max

File: 485-max-consecutive-ones/max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
         int size=nums.length;
         int count=0;
         int max=0;

         for(int i=0;i<size;i++){
              if(nums[i]!=1){
                max=Math.max(count,max);
                count=0;
              }else{

                 count++;

              }

             
         }

         max=Math.max(count,max);

         return max;
    }
}